# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../aufs/compile.patch.cross
# Copyright (C) 2007 - 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./util/Makefile.orig	2008-09-22 05:52:00.000000000 +0200
+++ ./util/Makefile	2008-12-12 09:48:35.000000000 +0100
@@ -24,11 +24,13 @@
 
 CFLAGS += -O2 -Wall ${EXTRA_CFLAGS} ${ccflags-y}
 
-.INTERMEDIATE: c2tmac.o c2tmac
-c2tmac.o: c2tmac.c ../include/linux/aufs_type.h
+.INTERMEDIATE: c2tmac
+c2tmac: c2tmac.c ../include/linux/aufs_type.h
+	$(HOSTCC) $(CFLAGS) c2tmac.c -o c2tmac
 
-.INTERMEDIATE: c2sh.o c2sh
-c2sh.o: c2sh.c ../include/linux/aufs_type.h
+.INTERMEDIATE: c2sh
+c2sh: c2sh.c ../include/linux/aufs_type.h
+	$(HOSTCC) $(CFLAGS) c2sh.c -o c2sh
 
 clean:
 	${RM} c2tmac c2sh aulchown aufs.5 etc_default_aufs *~ .*~
